src/styles.css

```
@@ -1,311 +1,487 @@
 }
 
-.pixel-name {
-  font-family: var(--pixel-font-display);
-  font-size: 2.5rem;
-  color: var(--pixel-fg);
-  margin-bottom: 1rem;
-  text-shadow: 2px 2px 0px var(--pixel-shadow);
+.metro-topbar__eyebrow {
+  margin: 0 0 0.35rem;
+  color: var(--retro-brown);
+  font-family: "SFMono-Regular", Menlo, Consolas, monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)